Snakefile

```diff
@@ -140,6 +140,7 @@ else:
 if config['enable'].get('build_natura_raster', False):        
     rule build_natura_raster:
         input: 
+            cutout_dir = "cutouts",
```

Comment:
Why is this necessary?

Reply:
because the cutout directory was hard coded in `build_natura_raster` before. Only like this the path can be set correctly by `mocksnakemake`

Reply:
Agreed, shouldn't be necessary.

Reply:
is reverted now